core/src/initializers/coreVersionValidator.ts

```diff
@@ -0,0 +1,20 @@
+import { Initializer } from "actionhero";
+
+export class CoreVersionValidator extends Initializer {
```

Comment:
Didn't seem like there was a great way to test this programmatically. I did it manually by switching the core version in the PG plugin and it worked great.

Reply:
Makes sense!